syntax = "proto3";

package yandex.cloud.priv.vpc.v1;

import "google/protobuf/empty.proto";
import "ydb/public/api/client/yc_private/quota/quota.proto";

option go_package = "github.com/ydb-platform/ydb/ydb/public/api/client/yc_private/vpc;vpc";
option java_outer_classname = "PQS";

service QuotaService {

  rpc Get (quota.GetQuotaRequest) returns (quota.Quota);

  rpc UpdateMetric (quota.UpdateQuotaMetricRequest) returns (google.protobuf.Empty);

  rpc BatchUpdateMetric (quota.BatchUpdateQuotaMetricsRequest) returns (google.protobuf.Empty);

  rpc GetDefault (quota.GetQuotaDefaultRequest) returns (quota.GetQuotaDefaultResponse);

}
